TUTORIAL.md

```diff
@@ -951,6 +951,18 @@ input CollectionInput {
 *Rule #21: Structure mutation inputs to reduce duplication, even if this
  requires relaxing requiredness constraints on certain fields.*
 
+As in the example above, `collectionUpdate` takes two arguments. The first
+selects the collection to update, and the second provides the update data. An
+alternative to this would be one argument, `CollectionInput!`, which would
+include the ID. In fact, [some older Shopify APIs follow this
+pattern](https://shopify.dev/docs/api/admin-graphql/2023-07/mutations/productUpdate).
+However, this  is no longer recommended, as it makes it difficult to determine
+which parts of the call relate to 'select', and which relate to 'update'.
```

Comment:
> In fact, [some older Shopify APIs follow this pattern](https://shopify.dev/docs/api/admin-graphql/2023-07/mutations/productUpdate).

I'd prefer to leave out the reference to a specific Shopify mutation because the rest of this tutorial doesn't mention any specifically, and while using a Shopify-ish schema example, is meant to be standalone and agnostic.


Here's my suggestion:

```suggestion
As in the example above, `collectionUpdate` takes two arguments. `collectionId`
selects the collection to update, and `collection` provides the update data. An
alternative to this would be a single merged `collection: CollectionInput!` argument, where
`CollectionInput` would have a nullable `id` input field. However, this makes it
difficult to determine which parts of the call relate to 'select', and which relate
to 'update' and is not recommended.
```